docs/installation-and-operation/remote-sync.md

```diff
@@ -180,12 +145,14 @@ Now that you have content in your repository, you can set up your production Met
 
    - Click "Save changes". Metabase will verify it can reach your repository. If the connection fails, verify your token has the appropriate permissions and hasn't expired.
 
-6. Pull changes and/or enable auto-sync (optional):
+6. Pull changes or enable auto-sync (optional):
```

Comment:
I think this needs some words. This is about choosing to enable autosync or not - like there isn't a radio button list where you pick "pull changes" or "enable autosync"

Maybe:

6. Choose how to update your production Metabase.
  To automatically pull changes from your main branch every five minutes,  toggle on "Auto-sync with Git". 
  If you don't toggle auto-sync, you'll need to pull changes manually from your chosen branch . You can pull changes either from this settings screen or by clicking the down arrow icon in next to the synced collection in the navigation sidebar